### DISCUSSION TITLE
[ERR] failed to send smtp notification for id: email : fromAddress missing from config URL

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/notify#107] [Q&A]
Notify Email is failing with below message.

[ERR] failed to send smtp notification for id: email : fromAddress missing from config URL

Below is the configuration setting.
 #cat .config/notify/provider-config.yaml                                                                                                    
smtp:
  - id: email
    smtp_server: smtp.gmail.com
    smtp_username: email@gmail.com
    smtp_password: password
    from_Address: email@gmail.com
    smtp_cc:
      - email@gmail.com
    smtp_format: "{{data}}"

**Comment by parrasajad** [comment]
Hi @1213311, please use `from_address` instead of `from_Address`
